### PR TITLE
Feature/celery tasks

### PR DESCRIPTION
**Pull request by ArtsiomSt**
I have a question: how is better to inform users about the result of celery tasks? For ex. we have algorithm that supplies cars from dealers and it is applied to all showrooms and for example one of them cant afford buying new cars(so this showroom does not have money to commit a transaction). Is it good idea to make for example SupplyHistory table and store such cases there?

### CODE REVIEW

**Comment by rupeq** [comment]
> I have a question: how is better to inform users about the result of celery tasks? For ex. we have algorithm that supplies cars from dealers and it is applied to all showrooms and for example one of them cant afford buying new cars(so this showroom does not have money to commit a transaction). Is it good idea to make for example SupplyHistory table and store such cases there?

yep, we can create a separate table for that. 
another approach can be sending email
or more difficult approaches like real time notifications (channels/sockets).

File: CarShowRoom/sellers/signals/handlers.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            try:
+                showroom_car = ShowroomCar.objects.get(
+                    Q(car_showroom=discount.car_showroom_id)
+                    & Q(car=instance.car_id)
+                    & ~Q(dealer=None)
+                )
+                dealercar_object = DealerCar.objects.get(
+                    Q(car=instance.car_id) & Q(dealer=showroom_car.dealer_id)
+                )
+                print(dealercar_object)
```
Comment (rupeq) [inline CarShowRoom/sellers/signals/handlers.py:37]: delete this

**Review by rupeq — APPROVED** [review]
overall lgtm. Can you record a demo how all these work?

**Comment by ArtsiomSt** [comment]
Okey, but I have to find something for recording)

File: CarShowRoom/sellers/signals/handlers.py (hunk 1 above)
Comment (ArtsiomSt) [reply CarShowRoom/sellers/signals/handlers.py:37]: Okey, missed it

**Pull request merged**